semcore/input-mask/src/InputMask.tsx

```diff
@@ -68,6 +69,8 @@ export type InputMaskValueProps = InputValueProps & {
   title?: string;
 
   includeInputProps?: string[];
+
+  maskOnlySymbols?: Record<string, boolean>;
```

Comment:
What it was added for?

Reply:
Just because for correct types

Reply:
Should we expose it?

Reply:
We use it in InputTrigger in DatePicker, so, yes

Reply:
Ok, so, please, add description and specify default value for better property documentation. Also we need to add corresponding changelog note

Reply:
added